python/src/opendp/context.py

```diff
@@ -243,11 +247,6 @@ def loss_of(epsilon=None, delta=None, rho=None, U=None) -> Tuple[Measure, Union[
     :param U: The type of the privacy parameter; Inferred if not provided.
 
     """
-    if epsilon is None and rho is None:
-        raise ValueError("Either epsilon or rho must be specified.")
-    if epsilon is None and delta is not None:
-        raise ValueError("Epsilon must be specified if delta is given.")
-
```

Comment:
Why remove these checks? It is valid for `rho` to be the only argument.

Reply:
The prompt for doing this is that the annotation additions in the earlier commit in the PR were causing mypy failures later in this function. The checks here are a little too complicated, and mypy was thinking that `None`s could come through. After replacing these with just one guard below, mypy is happy.